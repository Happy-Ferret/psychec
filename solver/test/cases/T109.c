void f()
{
    T1 t1;
    t1 = 0; // Pointer or integer (defaults to int)

    T2 t2;
    t2 = 5; // An integer.
}
