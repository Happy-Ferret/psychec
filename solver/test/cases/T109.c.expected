/* Forward declarations */

/* Type definitions */
typedef  int T2 ;
typedef  int /*<<< orphan or scalar*/ T1 ;

/* Variables */

